ES/ESmy022.xml

```diff
@@ -30,226 +30,359 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
+                                              on <locus from="50r" to="51r"/> the scribe reduced the size of the script
+                                              and accomodated on the pages
+                                              more written lines than the ruling allows.
+                                              It cannot be excluded that more than one scribe participated in copying.
+                                               </desc>
+                                         </handNote>
+                                      </handDesc>
+
+
+                                      <decoDesc>
+                                        <decoNote type="frame" xml:id="d1">
+                                           <locus target="#8r"/>:
+                                           <desc>Crude bichrome (black, red) ornamental
+                                           band.</desc>
+                                         </decoNote>
+
+                                         <decoNote type="drawing" xml:id="d2">
+                                            <locus target="#52v"/>
+                                            <desc>A figure of an equestrian saint, crudely drawn in pencil, poorly visible.</desc>
+                                         </decoNote>
+                                       </decoDesc>
+
+                                      <additions>
+                                         <list>
+                                            <item xml:id="a1">
+                                               <locus target="#1r"/>
+                                               <desc type="GuestText"><title type="complete" ref="LIT4145Onausp" cert="medium"/>
+                                               The ink of the note is abraded or faded off,
+                                                 the text is hardly readable.</desc>
+                                            </item>
+
+                                            <item xml:id="e1">
+                                              <locus target="#1v"/>
+                                              <desc>Pen trials and doodles in ink,
+                                                in different hands, some written upside down.</desc>
+                                            </item>
+                                            <item xml:id="e2">
+                                              <locus target="#8v #9r #11v #16v #27r #52r"/>
+                                              <desc>Recent doodles in pencil.</desc>
+                                            </item>
+
+                                            <item xml:id="e3">
+                                               <desc type="findingAid">Notes in the upper margin indicate
+                                                  indicate readings of <ref target="#p2_i1"/> assigned to the
```

Comment:
I would replace the ref target to p2 with the name of the work, to make the text readable (readings of the Mashafa gessew)

Reply:
Ok